create_derivatives/helpers.py

```diff
@@ -5,3 +5,27 @@ def check_dir_exists(dir):
     if not Path(dir).is_dir():
         raise Exception("Expected directory {} does not exist".format(dir))
     return True
+
+
+def matching_files(directory, prefix=None, suffix=None,
+                   prepend=False):
+    """Get a list of files that start with a specific prefix, optionally removing
+    any files that end in `_001`.
+
+    Args:
+        directory (str): The directory containing files.
+        prefix (str): A prefix to match filenames against.
+        suffix (str): A suffix (file extension) to match filenames against.
+        skip (bool): Flag indicating if files ending with `_001` should be removed.
+        prepend (bool): Add the directory to the filepaths returned
+    Returns:
+        files (lst): a list of files that matched the identifier.
+    """
```

Comment:
Remove references to the skip functionality.

Reply:
This will happen when I merge into development so I'm going to leave it alone for now